annotations/fenix/metrics/recent_tabs.in_progress_media_tab_opened/README.md

```diff
@@ -0,0 +1,8 @@
+---
+tags:
+  - RecentTabs
+---
+
+See https://github.com/mozilla-mobile/fenix/pull/20138
+Event generated when the user has opened a tab with in progress media from home screen.
```

Comment:
Not really sure what this file description should contain. Is this appropriate or should we include more details?

Reply:
This seems redundant with the information in the probe. You can see the [contributing guidelines](https://mozilla.github.io/glean-annotations/contributing/) for what should be added in this section. If you can't think of anything yet, you can just put this in (what I used for the initial set):

```
This is a stub commentary for the `addons.has_installed_addons` metric: please feel free to edit (read the
[contributing guidelines](https://github.com/mozilla/glean-annotations/blob/main/CONTRIBUTING.md)
if you haven't done this before)
```